### PR TITLE
CTLIB: ct_con_props() set/get client_charset

### PR DESCRIPTION
**Pull request by alltilla**
I wanted to set the client side character encoding from libdbi, but I could not find a proper setter/getter API for it. I saw, that similar options were set/got in ct_con_props() so I decided to add it there.

Signed-off-by: Attila Szakacs <attila.szakacs@balabit.com>

### CODE REVIEW

File: include/cspublic.h [hunk 1]
```diff
@@ -248,6 +248,8 @@ enum
 #define CS_SEC_APPDEFINED CS_SEC_APPDEFINED
 	CS_STICKY_BINDS = 9151,
 #define CS_STICKY_BINDS CS_STICKY_BINDS
+	CS_CLIENTCHARSET = 9152,
```
Comment (freddy77) [inline include/cspublic.h]: I think would be better as 9300, Sybase is using 9152 for CS_HAVE_CMD

**Comment by freddy77** [comment]
Looks clean enough to be merged in next release.
Only comment is the number, can you update and move the constant after the 9300 one?

File: include/cspublic.h (hunk 1 above)
Comment (alltilla) [reply include/cspublic.h]: Sure thing, thanks for the quick reply :)

**Comment by freddy77** [comment]
Merged, did couple of fixes

**Pull request closed without merging**